code/common/src/test/scala/com/avast/cloud/datadog4s/inmemory/Record.scala

```diff
@@ -0,0 +1,5 @@
+package com.avast.cloud.datadog4s.inmemory
+
+import com.avast.datadog4s.api.Tag
+
+case class Record[A](value: A, tags: Seq[Tag])
```

Comment:
`Set` instead of `Seq`?

Reply:
we are using `Tag*` in the code ATM - the implementation of this should match that